packages/barcode-scanning/src/definitions.ts

```diff
@@ -505,27 +520,510 @@ export interface Barcode {
    * @example "CapacitorJS"
    */
   displayValue: string;
+  /**
+   * Driver license or ID card.
+   *
+   * @since 7.0.0
+   * @example { firstName: 'John', lastName: 'Doe', licenseNumber: '1234567890' }
+   */
+  driverLicense?: DriverLicense;
+  /**
+   * An email message from a 'MAILTO:'.
+   *
+   * @since 7.0.0
+   * @example { address: '', body: '', subject: '', type: 2 }
+   */
+  email?: Email;
   /**
    * The barcode format.
    *
    * @since 0.0.1
    * @example "QR_CODE"
    */
   format: BarcodeFormat;
+
+  /**
+   * GPS coordinates from a 'GEO:'.
+   *
+   * @since 7.0.0
+   * @example { latitude: 37.422004, longitude: -122.084095 }
+   */
+  geoPoint?: GeoPoint;
+  /**
+   * Phone number info.
+   *
+   * @since 7.0.0
+   * @example { number: '+1234567890', type: 2 }
+   */
+  phone?: Phone;
   /**
    * The barcode value in a machine readable format.
    *
    * @since 0.0.1
    * @example "CapacitorJS"
    */
   rawValue: string;
+  /**
+   * A sms message from a 'SMS:'.
+   *
+   * @since 7.0.0
+   * @example { phoneNumber: '+1234567890', message: 'Hello, Capawesome!' }
+   */
+  sms?: Sms;
+  /**
+   * A URL and title from a 'MEBKM:'.
+   *
+   * @since 7.0.0
+   * @example { url: 'https://capacitorjs.com', title: 'CapacitorJS' }
+   */
+  urlBookmark?: UrlBookmark;
   /**
    * The barcode value type.
    *
    * @since 0.0.1
    * @example "TEXT"
    */
   valueType: BarcodeValueType;
+  /**
+   * A wifi network parameters from a 'WIFI:'.
+   *
+   * @since 7.0.0
+   * @example { encryptionType: 3 ,password: 'CapacitorJS', ssid: 'CapacitorJS' }
+   */
+  wifi?: BarcodeWifi;
```

Comment:
```suggestion
  wifi?: Wifi;
```

My initial idea was to prefix all objects with `Barcode` but after seeing your changes I like this approach more. ^^